includes/xh-blank.conf

```diff
@@ -0,0 +1,7 @@
+location /blank {
+	return 200 <!DOCTYPE html>
+}
+
+location /blank/ {
+	return 200 <!DOCTYPE html>
+}
```

Comment:
Great that this could be done so tightly here. Agree re. suggested change to `/xhBlank/`.

We should include in the commented-out sample app conf code within xh.conf and note in the CL that apps should add to their app files in order for this to have an effect.  

(I don't know if there is a better way we could be packaging this stuff. )